### DISCUSSION TITLE
Implement multi column sorting in clarity server driven datagrid

### DISCUSSION THREAD

**Discussion by vannguyen48** [vmware/clarity#5913] [Q&A]
I'm trying to implement multi column sort using clarity server driven datagrid. The library currently supports sorting on 1 column at a time, and not multiple columns. I was wondering if you have resources to share or ideas I can try out. Thanks!

**Comment by ph55** [comment]
Not quite possible right now.
I've opened feature requests and they were "captured": https://github.com/vmware/clarity/issues/1067 , https://github.com/vmware/clarity/issues/1468

It's good to see other people request this feature. Must have in any enterprise datagrid IMO.
